python/cudf/cudf/tests/test_orc.py

```diff
@@ -318,6 +319,31 @@ def test_orc_read_rows(datadir, skiprows, num_rows):
     np.testing.assert_allclose(pdf, gdf)
 
 
+def test_orc_read_skiprows(tmpdir):
+    fname = tmpdir.join("TestOrcFile.skiprows.orc")
```

Comment:
```suggestion
    bytes_obj = BytesIO()
```

We can directly write to a BytesIO object instead of temp dir here